core/tracing/hooks.go

```diff
@@ -171,6 +183,7 @@ type (
 )
 
 type Hooks struct {
+	Backend Backend
```

Comment:
Not sure I understand why the hooks struct should expose Backend??

Reply:
Sorry, not fully removed